test/test-pages/001/expected-metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Get your Frontend JavaScript Code Covered | Code",
+  "byline": "Nicolas Perriault —",
+  "excerpt": "width=device-width"
```

Comment:
Doh: https://github.com/mozilla/readability/pull/36

This code was originally written with `let`, but this broke when we switched everything to `var`.